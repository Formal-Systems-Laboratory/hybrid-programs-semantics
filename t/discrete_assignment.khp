vars a ; a := 0
