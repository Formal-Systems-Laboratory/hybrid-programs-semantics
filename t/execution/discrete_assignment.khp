vars a ; a := 0.0 + 0.1
