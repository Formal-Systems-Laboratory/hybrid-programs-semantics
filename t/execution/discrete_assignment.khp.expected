<generatedTop>
  <k>
    .
  </k>
  <pgmVars>
    a , .Ids
  </pgmVars>
  <state>
    a |-> 0.0 + 0.1
  </state>
  <evolutionConditions>
    .Set
  </evolutionConditions>
  <nonDetAssignments>
    .Map
  </nonDetAssignments>
  <counter>
    0
  </counter>
  <constraints>
    .
  </constraints>
</generatedTop>
